src/containers/BuySellCrypto/BuyCrypto/BuyCrypto.js

```diff
@@ -254,20 +260,15 @@ class BuyCrypto extends Component {
    * @param {Number} amount Amount to get price for
    * @param {Boolean} fromFiat (Optional, Default = false) Whether to convert from fiat to crypto instead of crypto to fiat
    */
-  getPrice = (amount, fromFiat = false) => {
-    const price = this.props.rates[this.state.toCurr]
+  getPrice = (amount) => {
```

Comment:
extract getPrice method in a separate utils file. You can add it to the same place where truncateDecimal is. We should strive to remove all logic outside of components